homeassistant/components/dsmr/config_flow.py

```diff
@@ -346,6 +348,12 @@ async def async_step_init(
                             CONF_TIME_BETWEEN_UPDATE, DEFAULT_TIME_BETWEEN_UPDATE
                         ),
                     ): vol.All(vol.Coerce(int), vol.Range(min=0)),
+                    vol.Optional(
+                        CONF_RECONNECT_INTERVAL,
```

Comment:
I suggest we don't add an option for this but just have a sane default. When does it need to be customized?

Reply:
I don't think it is neccessary to be able to customize it.

Reply:
I only made it an option because that's what it used to be, a sane default seems good to me too. I guess this just becomes a one line fix then?